{
  "Software_Repo_MappingId": null,
  "Software_RepoId": "f6b863cd-8f1c-4583-8bc5-ee59e86e5cf1",
  "Software_Name": "Python 3.12.5 (64-bit)",
  "CVEID": "CVE-2017-17522",
  "Summary": "Lib/webbrowser.py in Python through 3.6.3 does not validate strings before launching the program specified by the BROWSER environment variable, which might allow remote attackers to conduct argument-injection attacks via a crafted URL. NOTE: a software maintainer indicates that exploitation is impossible because the code relies on subprocess.Popen and the default shell=False setting",
  "LastUpdate": 0,
  "CVEE": {
    "CVEEId": "CVE-2017-17522",
    "CVSS_V2_baseScore": 6.8,
    "CVSS_V2_Data": "{\"version\":\"2.0\",\"baseScore\":6.8,\"accessVector\":\"NETWORK\",\"vectorString\":\"AV:N/AC:M/Au:N/C:P/I:P/A:P\",\"authentication\":\"NONE\",\"integrityImpact\":\"PARTIAL\",\"accessComplexity\":\"MEDIUM\",\"availabilityImpact\":\"PARTIAL\",\"confidentialityImpact\":\"PARTIAL\",\"scope\":null,\"attackVector\":null,\"baseSeverity\":null,\"userInteraction\":null,\"attackComplexity\":null,\"privilegesRequired\":null}",
    "CVSS_V3_baseScore": 8.8,
    "CVSS_V3_Data": "{\"version\":\"3.0\",\"baseScore\":8.8,\"accessVector\":null,\"vectorString\":\"CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H\",\"authentication\":null,\"integrityImpact\":\"HIGH\",\"accessComplexity\":null,\"availabilityImpact\":\"HIGH\",\"confidentialityImpact\":\"HIGH\",\"scope\":\"UNCHANGED\",\"attackVector\":\"NETWORK\",\"baseSeverity\":\"HIGH\",\"userInteraction\":\"REQUIRED\",\"attackComplexity\":\"LOW\",\"privilegesRequired\":\"NONE\"}",
    "PublishedDate": "2017-12-14T16:29:00.713",
    "ModifyDate": "2024-11-21T03:18:05.55",
    "VulnStatus": "Modified",
    "JsonData": "{\"id\": \"CVE-2017-17522\", \"summary\": \"Lib/webbrowser.py in Python through 3.6.3 does not validate strings before launching the program specified by the BROWSER environment variable, which might allow remote attackers to conduct argument-injection attacks via a crafted URL. NOTE: a software maintainer indicates that exploitation is impossible because the code relies on subprocess.Popen and the default shell=False setting\", \"created_at\": \"2017-12-14T16:29:00Z\", \"updated_at\": \"2024-11-21T03:18:05Z\", \"cvss\": {\"v2\": 6.8, \"v3\": 8.8}, \"vendors\": {\"python\": [\"python\"]}, \"cwes\": [\"CWE-74\"], \"raw_nvd_data\": {\"id\": \"CVE-2017-17522\", \"cveTags\": [{\"tags\": [\"disputed\"], \"sourceIdentifier\": \"cve@mitre.org\"}], \"metrics\": {\"cvssMetricV2\": [{\"type\": \"Primary\", \"source\": \"nvd@nist.gov\", \"cvssData\": {\"version\": \"2.0\", \"baseScore\": 6.8, \"accessVector\": \"NETWORK\", \"vectorString\": \"AV:N/AC:M/Au:N/C:P/I:P/A:P\", \"authentication\": \"NONE\", \"integrityImpact\": \"PARTIAL\", \"accessComplexity\": \"MEDIUM\", \"availabilityImpact\": \"PARTIAL\", \"confidentialityImpact\": \"PARTIAL\"}, \"acInsufInfo\": false, \"impactScore\": 6.4, \"baseSeverity\": \"MEDIUM\", \"obtainAllPrivilege\": false, \"exploitabilityScore\": 8.6, \"obtainUserPrivilege\": false, \"obtainOtherPrivilege\": false, \"userInteractionRequired\": true}], \"cvssMetricV30\": [{\"type\": \"Primary\", \"source\": \"nvd@nist.gov\", \"cvssData\": {\"scope\": \"UNCHANGED\", \"version\": \"3.0\", \"baseScore\": 8.8, \"attackVector\": \"NETWORK\", \"baseSeverity\": \"HIGH\", \"vectorString\": \"CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H\", \"integrityImpact\": \"HIGH\", \"userInteraction\": \"REQUIRED\", \"attackComplexity\": \"LOW\", \"availabilityImpact\": \"HIGH\", \"privilegesRequired\": \"NONE\", \"confidentialityImpact\": \"HIGH\"}, \"impactScore\": 5.9, \"exploitabilityScore\": 2.8}]}, \"published\": \"2017-12-14T16:29:00.713\", \"references\": [{\"url\": \"http://www.securityfocus.com/bid/102207\", \"tags\": [\"Third Party Advisory\", \"VDB Entry\"], \"source\": \"cve@mitre.org\"}, {\"url\": \"https://bugs.python.org/issue32367\", \"source\": \"cve@mitre.org\"}, {\"url\": \"https://security-tracker.debian.org/tracker/CVE-2017-17522\", \"tags\": [\"Issue Tracking\", \"Third Party Advisory\"], \"source\": \"cve@mitre.org\"}, {\"url\": \"http://www.securityfocus.com/bid/102207\", \"tags\": [\"Third Party Advisory\", \"VDB Entry\"], \"source\": \"af854a3a-2127-422b-91ae-364da2661108\"}, {\"url\": \"https://bugs.python.org/issue32367\", \"source\": \"af854a3a-2127-422b-91ae-364da2661108\"}, {\"url\": \"https://security-tracker.debian.org/tracker/CVE-2017-17522\", \"tags\": [\"Issue Tracking\", \"Third Party Advisory\"], \"source\": \"af854a3a-2127-422b-91ae-364da2661108\"}], \"vulnStatus\": \"Modified\", \"weaknesses\": [{\"type\": \"Primary\", \"source\": \"nvd@nist.gov\", \"description\": [{\"lang\": \"en\", \"value\": \"CWE-74\"}]}], \"descriptions\": [{\"lang\": \"en\", \"value\": \"Lib/webbrowser.py in Python through 3.6.3 does not validate strings before launching the program specified by the BROWSER environment variable, which might allow remote attackers to conduct argument-injection attacks via a crafted URL. NOTE: a software maintainer indicates that exploitation is impossible because the code relies on subprocess.Popen and the default shell=False setting\"}, {\"lang\": \"es\", \"value\": \"** EN DISPUTA ** Lib/webbrowser.py en Python hasta la versi\\u00f3n 3.6.3 no valida las cadenas antes de iniciar el programa especificado por la variable de entorno BROWSER. Esto podr\\u00eda permitir que atacantes remotos lleven a cabo ataques de inyecci\\u00f3n de argumentos mediante una URL manipulada. NOTA: el mantenedor del software indica que es imposible la explotaci\\u00f3n de esta vulnerabilidad debido a que el c\\u00f3digo conf\\u00eda en subprocess.Popen y el ajuste por defecto shell=False.\"}], \"lastModified\": \"2024-11-21T03:18:05.550\", \"configurations\": [{\"nodes\": [{\"negate\": false, \"cpeMatch\": [{\"criteria\": \"cpe:2.3:a:python:python:*:*:*:*:*:*:*:*\", \"vulnerable\": true, \"matchCriteriaId\": \"6BC31A8A-1EA3-4C8C-832F-3C2E5A8F5B69\", \"versionEndIncluding\": \"3.6.3\"}], \"operator\": \"OR\"}]}], \"sourceIdentifier\": \"cve@mitre.org\"}}\n",
    "LastUpdate": 0,
    "VendorL": [
      "python"
    ],
    "ProductL": [
      "python"
    ],
    "Software_Repo_Mappings": []
  },
  "Software_Repo_Mapping_Conditions": [
    {
      "Software_Repo_Mapping_ConditionId": 0,
      "Software_Repo_MappingId": null,
      "Criteria": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
      "VersionEndExcluding": null,
      "VersionEndIncluding": "3.6.3",
      "VersionStartExcluding": null,
      "VersionStartIncluding": null,
      "Vulnerable": true,
      "Software_Repo_Mapping": null
    }
  ]
}